certified-connectors/DQ on Demand/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7276 @@
+            "description": "Successful Request.",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "description": "The HTTP Status code of the response.",
+                  "type": "integer",
+                  "x-ms-visibility": "advanced"
+                },
+                "message": {
+                  "description": "A message describing the request status.",
+                  "type": "string",
+                  "x-ms-visibility": "advanced"
+                },
+                "data": {
+                  "description": "The response.",
+                  "type": "object",
+                  "properties": {
+                    "dqLine1": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address Line1.",
+                      "x-nullable": true
+                    },
+                    "dqLine2": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address Line2.",
+                      "x-nullable": true
+                    },
+                    "dqLine3": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address Line3.",
+                      "x-nullable": true
+                    },
+                    "dqLine4": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address Line4.",
+                      "x-nullable": true
+                    },
+                    "dqCity": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address City.",
+                      "x-nullable": true
+                    },
+                    "dqCounty_District": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address County / District.",
+                      "x-nullable": true
+                    },
+                    "dqState_Province": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address State / Province.",
+                      "x-nullable": true
+                    },
+                    "dqziP_PostalCode": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address ZIP / PostalCode.",
+                      "x-nullable": true
+                    },
+                    "dqCountry": {
+                      "x-ms-visibility": "important",
+                      "type": "string",
+                      "description": "Address CountryName.",
```

Comment:
@lukeswanton All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)